conf/messages.cy

```diff
@@ -1058,6 +1058,17 @@ available.requestvaluation.startCheckTab.option6.content=Mae penderfyniad llys y
 available.requestvaluation.startCheckTab.button.title=Parhau
 available.requestvaluation.startCheckTab.checkType.error.missing=Dewisiwch beth rydych am ei ddweud wrthym
 
+available.requestvaluation.listYear.2017.p1=Ni allwch ddweud wrthym bellach am newidiadau i fanylion eiddo ar gyfer unrhyw brisiadau ar restr ardrethu 2017.
+available.requestvaluation.listYear.2017.p2=Os ydych yn credu fod y gwerth ardrethol yn rhy uchel, gallwch anfon achos Herio atom. Mae’n rhaid eich bod wedi anfon achos Gwirio atom cyn 1 Ebrill 2023 ac (mae unrhyw un o’r rhain yn berthnasol):
+available.requestvaluation.listYear.2017.li1=eich bod yn anfon yr achos Herio o fewn 4 mis i'n penderfyniad ar yr achos Gwirio
+available.requestvaluation.listYear.2017.li2=rydych wedi aros mwy na 12 mis a heb dderbyn ein penderfyniad ar yr achos Gwirio
+available.requestvaluation.listYear.2017.li3=mae'r achos Herio yn ymwneud � newid yn yr ardal leol (er enghraifft gwaith ffordd hirdymor), rydych yn ei anfon o fewn 16 mis i anfon yr achos Gwirio a’ch bod wedi derbyn ein penderfyniad ar yr achos Gwirio
+available.requestvaluation.listYear.2017.p3=Gallwch hefyd anfon achos Herio atom os (mae unrhyw un o'r rhain yn berthnasol):
+available.requestvaluation.listYear.2017.li4=rydym yn newid y prisiad hwn ar �l 1 Ebrill 2023 ac o fewn 6 mis i ddyddiad y newid, rydych
+available.requestvaluation.listYear.2017.li5=effeithiodd benderfyniad llys ar werth ardrethol yr eiddo hwn a chyn 1 Hydref 2023, rydych
+available.requestvaluation.listYear.2017.link.text=yn anfon achos Gwirio o'r prisiad cyfredol
+available.requestvaluation.listYear.isDraft.p=Ni fydd y prisiad manwl hwn ar gael tan 1 Ebrill 2023. Byddwch yn gallu gwneud cais amdano o’r dyddiad hwn ymlaen.

```

Comment:
I spy some euros in here - is that happening on your machine too or is github just handling the charset weirdly?

Reply:
Looks fine in inteliji, I wonder if its github struggling with the encoding for punctuation

Reply:
If it looks OK on the screens then it's fine by me 👍